nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -83,6 +89,7 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
         self.original_checkpointing_granularity = base_module.language_model.encoder.activations_checkpoint_granularity
         self.original_checkpointing_num_layers = base_module.language_model.encoder.activations_checkpoint_num_layers
         self.original_checkpointing_method = base_module.language_model.encoder.activations_checkpoint_method
+        self.virtual_tokens = 0
```

Comment:
do you have to have the virtual tokens here? why not read it from the cfg like the `tokens_to_generate`?